packages/devtools-launchpad/webpack.config.js

```diff
@@ -145,16 +144,5 @@ module.exports = (webpackConfig, envConfig) => {
     webpackConfig = require("./webpack.config.devtools")(webpackConfig, envConfig);
   }
 
-  // NOTE: This is only needed to fix a bug with chrome devtools' debugger and
-  // destructuring params https://github.com/devtools-html/debugger.html/issues/67
-  if (getValue("transformParameters")) {
-    webpackConfig.module.rules.forEach(spec => {
-      if (spec.isJavaScriptLoader) {
-        const idx = spec.rules.findIndex(loader => loader.includes("babel-loader"));
-        spec.rules[idx] += "&plugins[]=transform-es2015-parameters";
-      }
-    });
-  }
-
```

Comment:
This is working anyway, because `spec.isJavascriptLoader` will always be `undefined` due to changes in webpack